### PR TITLE
Update mac-device-control-overview.md

### LINKED ISSUE #12500: Both Android (portableDevice) and iOS (appleDevice)  are not GA now.
[Enter feedback here]
Both Android (portableDevice) and iOS (appleDevice)  are not GA now.
Is it possible to add above?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1045c931-5e47-95f1-1851-4710cf59325e
* Version Independent ID: 811f41b4-b663-06db-9737-d871767065fb
* Content: [Device control for macOS](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/mac-device-control-overview?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/mac-device-control-overview.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/mac-device-control-overview.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about Portaabledevice and Appledevice

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12500

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/mac-device-control-overview.md [hunk 1]
```diff
@@ -104,6 +104,9 @@ Here are the properties you can use when you create the group and policy.
 
 ### Settings
 
+> [!NOTE]
+> Both Android (portableDevice) and iOS (appleDevice) are in Preview and not in GA now.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-device-control-overview.md]:
```suggestion
> Both Android (portableDevice) and iOS (appleDevice) are in public preview and not in general availability at the moment.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**